src/backend/gl/src/info.rs

```diff
@@ -689,6 +690,13 @@ pub(crate) fn query_all(
     {
         features |= Features::INDEPENDENT_BLENDING;
     }
+    if info.is_supported(&[
+        Es(3, 0),
+        Ext("WEBGL_compressed_texture_s3tc"),
```

Comment:
Currently, extensions list is empty for `Webgl`.

https://github.com/gfx-rs/gfx/blob/0a0cdaa39a1a84873aa7b195bdf2ba7fd3df478e/src/backend/gl/src/info.rs#L302-L304

Reply:
Oh I see. It worked for me because the `is_supported` is an "or" of passed requirements, and `Es 3.0` is always satisfied. That means this code is indeed wrong. I need to populate the extension list.

Reply:
First we need to expose extension list from `glow` somehow. Or copy-paste extensions checking from `glow` codebase:
https://github.com/grovesNL/glow/blob/7775e74363877347b2210d91277646a09d70dd08/src/web_sys.rs#L105-L109

Reply:
I've exposed the extensions in https://github.com/grovesNL/glow/pull/165. Let's wait for it to be in.